pkgs/development/libraries/audio/libgme/default.nix

```diff
@@ -21,4 +21,12 @@ in stdenv.mkDerivation {
   };
 
   buildInputs = [ cmake ];
+
+  # It used to reference it, in the past, but thanks to the postFixup hook, now
+  # it doesn't
+  disallowedReferences = [ gcc-unwrapped ];
```

Comment:
You could also use `stdenv.cc.cc` to get your hands onto the unwrapped compiler. A bit clearer in my opinion and in theory doesn't depend on knowing what stdenv variant is being used (default stdenv, `gcc8Stdenv`, `clangStdenv` etc.).

Reply:
Good idea, I was looking for this attribute but I didn't find it. I've pushed this and I'm waiting to see if it'll builds on Darwin.